curations/nuget/nuget/-/StackExchange.Redis.yaml

```diff
@@ -42,24 +42,77 @@ revisions:
   1.2.6:
     licensed:
       declared: MIT
+  2.0.0-alpha.465:
+    licensed:
+      declared: MIT
   2.0.495:
     licensed:
       declared: MIT
+  2.0.510:
+    licensed:
+      declared: MIT
   2.0.513:
     licensed:
       declared: MIT
   2.0.519:
     licensed:
       declared: MIT
+  2.0.544:
+    licensed:
+      declared: MIT
+  2.0.558:
+    described:
+      releaseDate: '2019-04-03'
+      sourceLocation:
+        name: stackexchange.redis
+        namespace: stackexchange
+        provider: github
+        revision: v2.0.558
+        type: git
+        url: 'https://github.com/stackexchange/stackexchange.redis/commit/v2.0.558'
+    licensed:
+      declared: MIT
   2.0.571:
     licensed:
       declared: MIT
+  2.0.588:
+    licensed:
+      declared: MIT
   2.0.593:
     licensed:
       declared: MIT
   2.0.600:
     licensed:
       declared: MIT
   2.0.601:
-    licensed:
-      declared: MIT
+    described:
+      sourceLocation:
+        name: stackexchange.redis
+        namespace: stackexchange
+        provider: github
+        revision: 4a0d3538da09fc1ddba120efaaefd5787533a1e0
+        type: git
+        url: 'https://github.com/stackexchange/stackexchange.redis/commit/4a0d3538da09fc1ddba120efaaefd5787533a1e0'
+    licensed:
+      declared: MIT
+  2.1.28:
+    described:
+      sourceLocation:
+        name: stackexchange.redis
+        namespace: stackexchange
+        provider: github
+        revision: 5c3036ab7aebd647f3dd3d89f07e02d4c2a48d5e
+        type: git
+        url: 'https://github.com/stackexchange/stackexchange.redis/commit/5c3036ab7aebd647f3dd3d89f07e02d4c2a48d5e'
```

Comment:
This commit goes to v2.1.30.

Reply:
Agree - I see the same thing on row 3 here https://github.com/stackexchange/stackexchange.redis/commit/5c3036ab7aebd647f3dd3d89f07e02d4c2a48d5e